src/esp/gfx/Drawable.h

```diff
@@ -110,7 +110,7 @@ class Drawable : public Magnum::SceneGraph::Drawable3D {
   uint64_t drawableId_;
 
   scene::SceneNode& node_;
-  Magnum::GL::Mesh& mesh_;
+  Magnum::GL::Mesh* mesh_ = nullptr;
```

Comment:
Can you post a reply here and explain to other reviewers why we did this change?

Reply:
The `createRenderer` flag prevents the function `GenericMeshData::uploadBuffersToGPU` from running, which is the function that creates `GL::Mesh`. This means that the `GenericMeshData` that are inserted into `ResourceManager::meshes_` will not have set `GL::Mesh` fields.

However, `getMagnumGLMesh` is still being called on elements of `meshes_`. These calls will return NULL because no `GL::Mesh` objects were ever created. Yet we are still trying to do things with the returned values, which is unsafe. 

Therefore, we instead store pointers to the `GL::Mesh` objects so that we must explicitly dereference them, which requires first checking that they are not `nullptr`. This is a cleaner way of handling errors.